environments/development.docker-compose.yaml

```diff
@@ -0,0 +1,66 @@
+version: '3.8'
+services:
+  db:
+    build: ../pipeline/postgres
+    environment:
+    - POSTGRES_DB=docker
+    - POSTGRES_USER=docker
+    - POSTGRES_PASSWORD=${OCAML_BENCH_DB_PASSWORD?required}
+    restart: always
+    ports: ["5432:5432"]
+  graphql-engine:
+    image: hasura/graphql-engine:v1.3.3
+    ports:
+    - "8080:8080"
+    depends_on:
+    - "db"
+    restart: always
+    environment:
+      HASURA_GRAPHQL_DATABASE_URL: postgres://docker:docker@db:5432/docker
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+      HASURA_GRAPHQL_ADMIN_SECRET: ${OCAML_BENCH_GRAPHQL_KEY?required}
+  frontend:
+    build: ../frontend
+    ports:
+    - "80:8082"
```

Comment:
```suggestion
    - "8082:80"
```

Reply:
Yes! `HOST:CONTAINER`.

I had some issues compiling the backend on arm64 so couldn't test it. It seems to work fine on the production machine though.

I also had issues with hasura constantly running out of mem and being OOM killed by docker.

I suspected that these are arm64 specific.

If you run into any issues while starting compose, please report them.